### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by OscarCamacho**
Builder Class generation now works!

### CODE REVIEW

File: src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+        addCommonFields();
+        if (annotatedClass.useFluentBuilder()) {
+            addFluentSetters();
+        } else {
+            addSimpleSetters();
+        }
+        addBuildMethod();
+    }
+
+    private void addCommonFields () {
```
Comment (Dniel3) [inline src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java:39]: delete extra space     private void addCommonFields() {

File: src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        this.getConstructors().add(privateConstructor);
+    }
+
+    private void addSimpleConstructor() {
+        ConstructorDescriptor publicConstructor = new ConstructorDescriptor(computeBuilderName());
+        publicConstructor.addModifier(PUBLIC);
+        this.getConstructors().add(publicConstructor);
+    }
+
+    private void addFluentSetters () {
```
Comment (Dniel3) [inline src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java:73]: delete extra space here and bellow occurrences

File: src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+                    .addModifier(PUBLIC)
+                    .addModifier(FINAL)
+                    .addArgument("value",
+                            attributeMethodMapping.getKey().getType())
+                    .addCodeLine(String.format("this.container.putIfAbsent(\"%s\", value);",
+                            attributeMethodMapping.getKey().getName())));
+        }
+    }
+
+    private void addBuildMethod () {
```
Comment (Dniel3) [inline src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java:103]: this method is too long. Consider adding two util methods one to generate build method when there is no constructor or has no args,  and the other when there is constructor with params

File: src/main/java/com/example/simpleannotation/model/GeneratedClass.java [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+    public List<String> getImplementClasses() {
+        return implementClasses;
+    }
+
+    public GeneratedClass setImplementClasses(List<String> implementClasses) {
+        this.implementClasses = implementClasses;
+        return this;
+    }
+
+    public List<String> getImports () {
```
Comment (Dniel3) [inline src/main/java/com/example/simpleannotation/model/GeneratedClass.java:67]: delete extra space  here and bellow

File: src/main/java/com/example/simpleannotation/processors/AbstractAnnotationProcessor.java [hunk 5]
```diff
@@ -32,16 +34,16 @@ public boolean process(Set<? extends TypeElement> annotations,
...
-                    printErrorMessage(baue.getMessage(), annotatedElement);
+                } catch (BadAnnotationUsageException | ClassGenerationException e) {
+                    printErrorMessage(e.getMessage(), annotatedElement);
                 }
         }
-        return true;
+        return false;
     }
 
     private void printErrorMessage (String errorMessage, Element element) {
```
Comment (Dniel3) [inline src/main/java/com/example/simpleannotation/processors/AbstractAnnotationProcessor.java:44]: delete extra space here and elsewhere

Comment (OscarCamacho) [reply src/main/java/com/example/simpleannotation/processors/AbstractAnnotationProcessor.java:44]: Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)

File: src/main/java/com/example/simpleannotation/model/GeneratedClass.java (hunk 4 above)
Comment (OscarCamacho) [reply src/main/java/com/example/simpleannotation/model/GeneratedClass.java:67]: Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)

File: src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java (hunk 3 above)
Comment (OscarCamacho) [reply src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java:103]: Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)

File: src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java (hunk 2 above)
Comment (OscarCamacho) [reply src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java:73]: Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)

File: src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java (hunk 1 above)
Comment (OscarCamacho) [reply src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java:39]: Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)

**Comment by OscarCamacho** [comment]
Thanks for the review @Dniel3!

**Pull request merged**